lib/krane/kubernetes_resource.rb

```diff
@@ -498,7 +511,7 @@ def validate_spec_with_kubectl(kubectl)
     end
 
     def validate_with_dry_run_option(kubectl, dry_run_option)
-      command = ["apply", "-f", file_path, dry_run_option, "--output=name"]
+      command = ["create", "-f", file_path, dry_run_option, "--output=name"]
```

Comment:
Can we use the result of `deploy_method` here?

Reply:
Not directly--`replace_force` isn't a real verb, and more importantly it looks like `replace` doesn't have a dry run flag at all.